tools/e2e/commands.ts

```diff
@@ -152,6 +153,15 @@ Cypress.Commands.add('clearRecordDrafts', () => {
   cy.reload()
 })
 
+Cypress.Commands.add('publishAndReload', () => {
+  cy.wait(1200)
+  cy.get('md-editor-publish-button').click()
+  cy.wait(1200)
```

Comment:
Waiting for the return of the publish request can and should be explicit:
```
cy.intercept({
        method: 'PUT',
        pathname: '**/records',
      }).as('insertRecord')
      cy.get('md-editor-publish-button').click()
      cy.wait('@insertRecord')
```

Reply:
On the other hand, yes we can try to get rid of the second `wait`.